src/lib/expert_partitioner/dialogs/format.rb

```diff
@@ -90,30 +80,32 @@ def create_dialog
       )
     end
 
+    def filesystem_items
+      [
+        Item(Id(Storage::FsType_EXT4), "Ext4"),
+        Item(Id(Storage::FsType_XFS), "XFS"),
+        Item(Id(Storage::FsType_BTRFS), "Btrfs"),
+        Item(Id(Storage::FsType_SWAP), "Swap"),
+        Item(Id(Storage::FsType_NTFS), "NTFS"),
+        Item(Id(Storage::FsType_VFAT), "VFAT")
+      ]
+    end
```

Comment:
JFYI: you can have translations even for constants, see https://github.com/webyast/webyast/wiki/Localization#translating-constants

*(I should move some parts this doc to YaST...)*